src/ray/protobuf/ray_client.proto

```diff
@@ -103,6 +103,12 @@ message PutRequest {
 message PutResponse {
   // The reference ID for the data that the server has stored.
   bytes id = 1;
+
+  // Whether or not the put was successful.
+  bool valid = 2;
+
```

Comment:
nit: Can we swap the order of `id` and `valid` to match `GetResponse` and `ClientTaskTicket` ?

Reply:
SGTM!

Reply:
ehh, actually I think we should keep it the same. As per Google's [protoguide](https://developers.google.com/protocol-buffers/docs/proto3#updating):
* Field numbers should not be modified

I'm not 100% sure this is necessary in this case (as we use the protocol date to ensure compatibility), but think it is better to follow this convention.

Reply:
okidoke, sounds good.